src/sdl2/mixer/mod.rs

```diff
@@ -250,6 +250,27 @@ impl Chunk {
         }
     }
 
+    /// Load chunk from a static byte buffer.
+    pub fn from_static_bytes(buf: &'static [u8]) -> Result<Chunk, String> {
+        let rw = unsafe {
+            sys::SDL_RWFromConstMem(buf.as_ptr() as *const c_void, buf.len() as c_int)
+        };
```

Comment:
I guess you could use `RWops` there (exactly `RWops::from_bytes(buf)`), I've seen other methods to do this and even make it a one-liner in `raw` declaration (avoiding declaring another var and doing another check I guess). I'm not sure about the benefits with that other than avoiding the use of another `unsafe` block (I'm not the Rust expert, so I might still be unfamiliar with that language design and quirks, same goes with SDL2).